src/InsetOrderOptimizer.h

```diff
@@ -40,7 +40,8 @@ class InsetOrderOptimizer
     z_seam_config(mesh.settings.get<EZSeamType>("z_seam_type"), mesh.getZSeamHint(), mesh.settings.get<EZSeamCornerPrefType>("z_seam_corner")),
     added_something(false),
     wall_overlapper_0(nullptr),
-    wall_overlapper_x(nullptr)
+    wall_overlapper_x(nullptr),
+    retraction_region_calculated(false)
```

Comment:
In general it's best to group boolean fields together so that they also get stored together. This helps with packing the class so that it'll use less memory and constructs faster.

Not really important in this case, but useful to know!